counsel.el

```diff
@@ -6522,6 +6527,23 @@ list as it may also be a build directory."
         (push (format "%s %s -o %s" cc f o) cmds)))
     cmds))
 
+;; Rust support
+;;
+;; We look for a Cargo.toml and is that exits offer the list of cargo
```

Comment:
is that exits -> if that exists?

But why not just consolidate this comment with the docstring?